cli/src/service.rs

```diff
@@ -355,84 +359,212 @@ pub fn new_light(config: Configuration) -> Result<impl AbstractService, ServiceE
-		let dummy_runtime = ::tokio::runtime::Runtime::new().unwrap();
-		let block_factory = |service: &<Factory as service::ServiceFactory>::FullService| {
-			let block_id = BlockId::number(service.client().chain_info().best_number);
-			let parent_header = service
-				.client()
-				.best_header(&block_id)
-				.expect("db error")
-				.expect("best block should exist");
-
-			futures::executor::block_on(service.transaction_pool().maintain(ChainEvent::NewBlock {
-				is_new_best: true,
-				id: block_id.clone(),
-				retracted: vec![],
-				header: parent_header,
-			}));
-
-			let consensus_net = ConsensusNetwork::new(service.network(), service.client().clone());
-			let proposer_factory = consensus::ProposerFactory {
-				client: service.client().clone(),
-				transaction_pool: service.transaction_pool().clone(),
-				network: consensus_net,
-				force_delay: 0,
-				handle: dummy_runtime.executor(),
-			};
-			let (proposer, _, _) = proposer_factory
-				.init(&parent_header, &validators, alice.clone())
-				.unwrap();
-			let block = proposer.propose().expect("Error making test block");
-			BlockImportParams {
-				origin: BlockOrigin::File,
-				justification: Vec::new(),
-				internal_justification: Vec::new(),
-				finalized: false,
-				body: Some(block.extrinsics),
-				storage_changes: None,
-				header: block.header,
-				auxiliary: Vec::new(),
-			}
-		};
-		let extrinsic_factory = |service: &SyncService<<Factory as service::ServiceFactory>::FullService>| {
-			let payload = (
-				0,
-				Call::GenericAsset(GenericAssetCall::transfer(
-					SPENDING_ASSET_ID,
-					RawAddress::Id(bob.public().0.into()),
-					69.into(),
-				)),
-				Era::immortal(),
-				service.client().genesis_hash(),
-			);
-			let signature = alice.sign(&payload.encode()).into();
-			let id = alice.public().0.into();
-			let xt = UncheckedExtrinsic {
-				signature: Some((RawAddress::Id(id), signature, payload.0, Era::immortal())),
-				function: payload.1,
-			}
-			.encode();
-			let v: Vec<u8> = Decode::decode(&mut xt.as_slice()).unwrap();
-			OpaqueExtrinsic(v)
-		};
+		let keystore_path = tempfile::tempdir().expect("Creates keystore path");
+		let keystore = sc_keystore::Store::open(keystore_path.path(), None).expect("Creates keystore");
+		let alice = keystore
+			.write()
+			.insert_ephemeral_from_seed::<sc_consensus_babe::AuthorityPair>("//Alice")
+			.expect("Creates authority pair");
+
+		let chain_spec = crate::chain_spec::dev::config();
+
+		// For the block factory
+		let mut slot_num = 1u64;
+
+		// For the extrinsics factory
+		let bob = Arc::new(AccountKeyring::Bob.pair());
+		let charlie = Arc::new(AccountKeyring::Charlie.pair());
+		let mut index = 0;
+
 		sc_service_test::sync(
-			sc_chain_spec::integration_test_config(),
-			|config| new_full(config),
-			|mut config| new_light(config),
-			block_factory,
-			extrinsic_factory,
+			chain_spec,
+			|config| {
+				let mut setup_handles = None;
```

Comment:
Not sure if CI/CD will auto fix it, have you run `cargo fmt` locally?

Reply:
Yes and it has passed the CI/CD formatting check

Reply:
Formatted code is not just easier developmwnt but also for better review experience. I suppose github code review formatting is based intents we submitted. May need to check with tech leads if we considered coding standards for further commits? cc @holygits